app/Http/Controllers/ProjectController.php

```diff
@@ -25,9 +27,7 @@ public function listProjects()
      */
     public function view(Project $project)
     {
-        $project->load('comments.user');
-
-        return view('project.view', ['project' => $project]);
+        return view('project.view', ['project' => $project, 'isUserFollower' => $project->isUserFollower(), 'followerCount' => $project->getFollowerCount()]);
```

Comment:
Yeah I wasn't to sure which way was the correct way to do this so I did it like this.